### ISSUE TITLE
Auto generate placeholder

### ISSUE THREAD

**Issue by nicdnepr** [yiisoft/yii2#15226]
If I want use placeholder with same label attribute need use code
```
<?= $form->field($model, 'customer_name')->textInput(['placeholder' => $model->getAttributeLabel('customer_name')])->label(false); ?>
```
Why no options to auto generate placeholder from AttributeLabel?

**Comment by developeruz** [comment]
Ups, @vladis84 it seems we did the same work :) I believe that this feature should be applied to all types of `Html::activeInput` like, for example, password field.

**Comment by vladis84** [comment]
You're right about that

**Comment by SilverFire** [comment]
Resolved by #15405
